video/out/cocoa_common.m

```diff
@@ -873,4 +883,16 @@ - (void)windowDidBecomeKey:(NSNotification *)notification
     [self didChangeMousePosition];
 }
 
+- (void)windowDidMiniaturize:(NSNotification *)notification
+{
+    struct vo_cocoa_state *s = self.vout->cocoa;
+    s->pending_events |= VO_EVENT_WIN_STATE;
+}
+
+- (void)windowDidDeminiaturize:(NSNotification *)notification
```

Comment:
Fixed in https://github.com/jaimeMF/mpv/commit/9896bb912e5afe97268b6481639a3aa6f9b8a3fa, I just pasted the block from the other method and forgot to update.

I don't mind using a single method, I was just using one for each of the delegate methods. Do you have some name suggestion for the method?

> Also, I think these should be added to the interface declaration in mpvadapter.h.

It's part of the NSWindowDelegate protocol, other methods (like `windowDidBecomeKey`) are not in the header file.